File: yunva17/프로그래머스/Lv2/피보나치_수.js
function solution(n) {
  let fibo = [];
  fibo[0] = 0;
  fibo[1] = 1;

  for (let i = 2; i <= n; i++) {
    fibo[i] = (fibo[i - 1] + fibo[i - 2]) % 1234567;
  }

  return fibo[n];
}

File: overthename/Programmers/level2/피보나치수.java
class Solution {
	public int solution(int n) {

		return fibo(n);
	}

	public int fibo(int n){
		int[] list = new int[n+1];
		list[0]=0;
		list[1]=1;

		for(int i=2;i<=n;i++){

			list[i]= (list[i-1]+list[i-2]) % 1234567;}

		return list[n] % 1234567;
	}
}